lightning-invoice/src/utils.rs

```diff
@@ -931,10 +946,23 @@ mod test {
 		];
 
 		let description_hash = crate::Sha256(Hash::hash("Description hash phantom invoice".as_bytes()));
-		let invoice = ::utils::create_phantom_invoice_with_description_hash::<EnforcingSigner,&test_utils::TestKeysInterface>(Some(payment_amt), None, 3600, description_hash, route_hints, &nodes[1].keys_manager, Currency::BitcoinTestnet).unwrap();
-
+		let non_default_invoice_expiry_secs = 4200;
+		let invoice = ::utils::create_phantom_invoice_with_description_hash::<
+			EnforcingSigner,
+			&test_utils::TestKeysInterface,
+		>(
+			Some(payment_amt),
+			None,
+			non_default_invoice_expiry_secs,
+			description_hash,
+			route_hints,
+			&nodes[1].keys_manager,
+			Currency::BitcoinTestnet,
```

Comment:
Let me know if this is a bit overkill for wrapping. Just modified what rustfmt did. I know it doesn't play too nicely with this repo in general so can wrap more like above.

Reply:
A good middle ground is to wrap at 100 characters as you are but also do so for parameter lists such that the parameters lists are packed into that width instead of having a parameter on each line. I'd leave off the trailing comma when doing so vs keeping it when there is one parameter per line.

Reply:
Cool. Will just fix this one up then to match the others :)